programmers/이상한-문자.js

```diff
@@ -0,0 +1,31 @@
+function solution(s) {
+
+    s = s.split(" ");
+    const sSpread = [...s];   
+    const len = sSpread.length;
+    //const sSpread = [...(s.split(" "))];  이것도 가능한것 같은데 어떤식으로 표현하는게 나을지 여쭤보고싶습니다.            
```

Comment:
두번째 방법이 더 간결하고 좋은  것 같아요
왜나면 코드가 길어지면 이런 한번에 할 수 있는 코드가 좀 더 보기 편하더라고요ㅎㅎ
물론 개인적인 의견입니다 참고만 해주세요

```suggestion
    s = s.split(" ");           
```
이렇게 해도 split의 반환값은 배열이라 원하시는 값을 가지실 수 있어요